docs/source/nn.md

```diff
@@ -505,47 +525,47 @@ Utility functions in other modules
     nn.utils.rnn.pack_sequence
     nn.utils.rnn.unpack_sequence
     nn.utils.rnn.unpad_sequence
+```
 
+```{eval-rst}
 .. autosummary::
     :toctree: generated
     :nosignatures:
     :template: classtemplate.rst
 
     nn.Flatten
     nn.Unflatten
+```
 
-Quantized Functions
---------------------
-
-Quantization refers to techniques for performing computations and storing tensors at lower bitwidths than
-floating point precision. PyTorch supports both per tensor and per channel asymmetric linear quantization. To learn more how to use quantized functions in PyTorch, please refer to the :ref:`quantization-doc` documentation.
-
-Lazy Modules Initialization
----------------------------
+## Quantized Functions
+Quantization refers to techniques for performing computations and storing tensors at lower bitwidths than floating point precision. PyTorch supports both per tensor and per channel asymmetric linear quantization. To learn more how to use quantized functions in PyTorch, please refer to the {ref}`quantization-doc` documentation.
 
+## Lazy Modules Initialization
+```{eval-rst}
 .. currentmodule:: torch
 .. autosummary::
     :toctree: generated
     :nosignatures:
     :template: classtemplate.rst
 
     nn.modules.lazy.LazyModuleMixin
+```
 
-Aliases
-_______
-
+## Aliases
 The following are aliases to their counterparts in ``torch.nn``:
-
+```{eval-rst}
 .. currentmodule:: torch
 .. autosummary::
     :toctree: generated
     :nosignatures:
     :template: classtemplate.rst
 
     nn.modules.normalization.RMSNorm
+```
+
+<!--
+This module needs to be documented. Adding here in the meantime for tracking purposes
 
-.. This module needs to be documented. Adding here in the meantime
-.. for tracking purposes
 .. py:module:: torch.nn.backends
```

Comment:
need to wrap this into {eval-rst} like [here](https://github.com/pytorch/pytorch/pull/154438/files#diff-f5f906033007a61905616c935edd01fd5d6f0d9c5e42d1a27601a826a19b915dR862).